src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -53,7 +53,7 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
             string strValue = value as string;
             if (strValue != null)
             {
-                return ColorConverterCommon.ConvertFromString(strValue, culture);
+                return ColorConverterCommon.ConvertFromString(strValue, culture ?? CultureInfo.CurrentCulture);
```

Comment:
Nit: it seems a bit more robust to me to put the defaulting inside ConvertFromSTring.